src/components/measureLanding/measureList/MeasureList.tsx

```diff
@@ -701,164 +702,29 @@ export default function MeasureList(props: {
-    setToastType("success");
-    setToastMessage("Measure exported successfully");
-    setDownloadState(warn ? "warning" : "success");
-    document.body.removeChild(link);
-  };
-
   const exportMeasure = async (elmErrorSeverity: string) => {
     setViewHumanReadableModal({
       open: false,
       measureId: "",
     });
-    setFailureMessage(null);
-    setDownloadState("downloading");
     try {
-      // we need to generate an abort controller for this call and bind it in the context of our ref
-      abortController.current = new AbortController();
-      const { ecqmTitle, model, version } = targetMeasure?.current ?? {};
-      const { status, data } = await measureServiceApi?.getMeasureExport(
-        targetMeasure.current?.id,
-        elmErrorSeverity,
-        abortController.current.signal
+      const measure: Measure = await measureServiceApi.fetchMeasure(
+        targetMeasure.current?.id
       );
-      const warn =
-        status === 201 && !targetMeasure?.current?.measureMetaData?.draft;
-      downloadZipFile(data, ecqmTitle, model, version, warn);
-    } catch (err) {
-      const errorStatus = err.response?.status;
-      if (err.message === "canceled") {
-        setToastOpen(false);
-        setDownloadState(null);
-      } else {
-        setToastType("danger");
-        setDownloadState("failure");
-        if (errorStatus === 409) {
-          const {
-            cql,
-            cqlErrors,
-            errors,
-            groups,
-            measureMetaData,
-            cqlLibraryName,
-            model,
-            baseConfigurationTypes,
-          } = await measureServiceApi?.fetchMeasure(targetMeasure.current?.id);
-          const missing = [];
-          if (_.isEmpty(cql)) {
-            missing.push("Missing CQL");
-          }
-          if (cqlErrors) {
-            missing.push("CQL Contains Errors");
-          }
-          if (!_.isEmpty(errors)) {
-            errors.forEach((error) => {
-              if (error.startsWith("MISMATCH_CQL_POPULATION_RETURN_TYPES")) {
-                missing.push("CQL Populations Return Types are invalid");
-              } else if (error.startsWith("MISMATCH_CQL_RISK_ADJUSTMENT")) {
-                missing.push("CQL Risk Adjustment are invalid");
-              } else if (error.startsWith("MISMATCH_CQL_SUPPLEMENTAL_DATA")) {
-                missing.push("CQL Supplemental Data Elements are invalid");
-              }
-            });
-          }
-          if (
-            (model.startsWith("QI-Core") &&
-              !/^(^[A-Z][a-zA-Z0-9]*$)/.test(cqlLibraryName)) ||
-            (model.startsWith("QDM") &&
-              !/^(^[A-Z][a-zA-Z0-9_]*$)/.test(cqlLibraryName)) ||
-            cqlLibraryName == null ||
-            cqlLibraryName.length > 64
-          ) {
-            missing.push("Measure CQL Library Name is invalid");
-          }
-          if (_.isEmpty(groups)) {
-            missing.push("Missing Population Criteria");
```

Comment:
Are these error messages being sent from somewhere else or are we moving forward with an umbrella error message for all of them?

Reply:
ooh I just moved to be in 1 place in exportUtil.ts We have 2 different export calls 1 from Measure list page and the other from Edit Measure page, The execution is exactly same so I just combined both into ExportUtil.ts